docs/repo/dependencies-node.md

```diff
@@ -36,3 +38,276 @@ internal sealed class MyDependenciesTreePropertiesProvider : IProjectTreePropert
     }
 }
 ```
+
+---
+
+## Handling removal of a dependency
+
+To handle removal of the "Remove" command on a dependency in the tree, export an implementation of `IProjectTreeActionHandler`. This contract exists in `UnconfiguredProject` scope, so you can import unconfigure project MEF parts if needed.
```

Comment:
Should highlight that this mechanism only applies to top-level dependencies, not transitive ones.